File: CODE FILES/Mobile/MyApiary/app/src/main/java/projects/myapiary/models/SensorsGetResponce.kt
package projects.myapiary.models

import com.google.gson.annotations.SerializedName

data class SensorsGetResponce(
    @SerializedName("id")
    val id: Int,
    @SerializedName("minValue")
    val minvalue: Float,
    @SerializedName("maxValue")
    val maxvalue: Float,
    @SerializedName("isWorking")
    val isworking: Boolean,
    @SerializedName("type")
    val type: String,
    @SerializedName("serialNumber")
    val serialnumber: String,
    @SerializedName("beehiveId")
    val beehiveId: Int,
    @SerializedName("baseStationName")
    val basestationname: String,
    @SerializedName("baseStationSerialNumber")
    val basestationserialnumber: String
)

File: CODE FILES/Backend/MyApiary/Resource.Api/Entities/SensorsGetResponce.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Resource.Api.Entities
{
    public class SensorsGetResponce
    {
        public int Id { get; set; }

        public float MinValue { get; set; }

        public float MaxValue { get; set; }

        public bool isWorking { get; set; }

        public string Type { get; set; }

        public string SerialNumber { get; set; }

        public int BeehiveId { get; set; }

        public string BaseStationName { get; set; }

        public string BaseStationSerialNumber { get; set; }
    }
}
